src/middlewared/middlewared/plugins/jail.py

```diff
@@ -265,9 +274,15 @@ def progress_callback(content):
 
         job.set_progress(0, start_msg)
         iocage.fetch(**options)
+
+        if options['name'] is not None:
+            for split_msg in job.progress['description'].split('\n'):
+                # This is to get the admin URL and such
+                fetch_output['install_notes'].append(split_msg)
```

Comment:
For future reference:

fetch_output['install_notes'] += job.progress['description'].split('\n')